src/dotnet/commands/dotnet-publish/PublishCommand.cs

```diff
@@ -207,6 +181,40 @@ private bool PublishProjectContext(ProjectContext context, string buildBasePath,
             return true;
         }
 
+        private Boolean InvokeBuildOnProject(ProjectContext context, String buildBasePath, String configuration)
```

Comment:
Good old extract method code generation :) 

---
In reply to: [57806699](https://github.com/dotnet/cli/pull/2124#discussion_r57806699) [](ancestors = 57806699)